.github/workflows/go.yml

```
@@ -2,9 +2,9 @@ name: Go
 
 on:
   push:
-    branches: [main]
+    branches: [main, release-1.2]
```

I am pretty sure if we use a filter here, we could have it account for future releases as well

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters

```suggestion
    branches: [main, release-**]
```